### DISCUSSION TITLE
🤖☁️ AWS Chatbot to Slack 💬

### DISCUSSION THREAD

**Discussion by Gary-H9** [ministryofjustice/analytical-platform#2581] [Firebreak Winter 23]
https://aws.amazon.com/about-aws/whats-new/2023/11/aws-chatbot-q-conversations-teams-slack/

This Slack App is [here](https://app.slack.com/apps-manage/T02DYEB3A/integrations/profile/A6L22LZNH/permissions), I think it would need to be configured further to make it usable. Unsure how this would function across multiple AWS accounts. MVP would focus on one AWS account.

**Comment by tom-webber** [comment]
Not pursuing as we're unsure how to work with authorisation / authentication for aws services. E.g. how do we know which aws account a user has permissions to query, and which one they're referring to in a query?
